packages/react-tabster/.storybook/preview.js

```diff
@@ -1,4 +1,3 @@
 import * as rootPreview from '../../../.storybook/preview';
 
 export const decorators = [...rootPreview.decorators];
-export const parameters = { layout: 'centered' };
```

Comment:
is this change ok ? ( I know migration generator replaces this file ), just to double check